### ISSUE TITLE
linter: jest/padding-around-after-all-blocks does not report missing trailing padding

### ISSUE THREAD

**Issue by hirunebot** [oxc-project/oxc#25590]
### What version of Oxlint are you using?

  1.78.0

  ### What command did you run?

  npx --yes oxlint@1.78.0 -c .oxlintrc.json reproduction.test.js

  ### What does your .oxlintrc.json (or oxlint.config.ts) config file look like?
```
  {
    "$schema": "./node_modules/oxlint/configuration_schema.json",
    "plugins": ["jest"],
    "env": {
      "es2021": true,
      "jest": true,
      "node": true
    },
    "rules": {
      "jest/padding-around-after-all-blocks": "error"
    }
  }
```
  ### What happened?

  Given the following `reproduction.test.js`:

  ```js
  afterAll(() => {});
  describe("suite", () => {});
  ```

  Oxlint exits successfully without reporting the missing blank line after `afterAll(...)`.

  ## Actual behavior

  No diagnostic is reported by `jest/padding-around-after-all-blocks`.

  ## Expected behavior

  The rule should report the missing trailing padding. The corrected code is:

  ```js
  afterAll(() => {});

  describe("suite", () => {});
  ```

  The upstream `eslint-plugin-jest` rule enforces padding both before and after an `afterAll` statement:

  - https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/padding-around-after-all-blocks.md
  - https://github.com/jest-community/eslint-plugin-jest/blob/main/src/rules/padding-around-after-all-blocks.ts

  Running the equivalent reproduction with `eslint-plugin-jest` reports the missing blank line before the statement following `afterAll(...)`.

  ## AI usage disclosure

  OpenAI Codex was used to help compare the behavior with the upstream rule, verify the reproduction against Oxlint 1.78.0, and draft this report. I reviewed and verified the reproduction and take responsibility for this report.
